metomi/isodatetime/data.py

```diff
@@ -320,24 +320,35 @@ class Duration(object):
 
     """Represent a duration or period of time.
 
+    Note that years, months, weeks and days are 'nominal' durations, whose
+    exact length of time depends on their position in the calendar. E.g., a
+    duration of 1 calendar year starts on a particular day of a particular
+    month and ends on the same day of the same month in the following calendar
+    year, and may be equal to 365 or 366 days in the Gregorian calendar.
+    Another example: a duration of 1 calendar day starts at a particular time
+    of day and ends at the same time of day the following calendar day, and
+    might be different to 24 hours if the you're using a local time zone that
+    can change due to daylight saving.
+
     Keyword arguments:
 
-    years (default 0): number of calendar years in the duration (an
-        inexact unit)
-    months (default 0): number of calendar months in the duration (also
-        an inexact unit)
-    weeks (default 0): number of weeks in the duration - cannot be
-        used in conjunction with other units (use multiples of 7 days
-        instead)
-    days (default 0): number of days in the duration
-    hours (default 0): number of hours in the duration
-    minutes (default 0): number of minutes in the duration
-    seconds (default 0): number of seconds in the duration
-    standardize (default False): boolean that, if True, switches on
-        adjusting the attributes so that small units have minimal values.
-        For example, 3664.4 seconds would become 1 hour, 1 minute, and 4.4
-        seconds. Attributes will not adjust for units that are inexact
-        (months and years).
+    years (int, default = 0): number of calendar years in the duration (an
```

Comment:
I have no idea what format this docstring was supposed to be in before, but it was definitely invalid!

> **FYI:** We are trying to pull towards Google Style docstrings as these can be read by Napoleon (the package as opposed to the historical figure) which allows us to autodocument.

```suggestion
    years (int): number of calendar years in the duration (an
```